platform/ios/src/MGLMapView.mm

```diff
@@ -189,10 +194,11 @@ @implementation MGLMapView
     BOOL _opaque;
 
     NS_MUTABLE_ARRAY_OF(NSURL *) *_bundledStyleURLs;
-    
+   
     MGLAnnotationContextMap _annotationContextsByAnnotationTag;
     /// Tag of the selected annotation. If the user location annotation is selected, this ivar is set to `MGLAnnotationTagNotFound`.
     MGLAnnotationTag _selectedAnnotationTag;
+    NS_MUTABLE_DICTIONARY_OF(NSString *, NSMutableArray *) *_annotationViewReuseQueueByIdentifier;
```

Comment:
The NSMutableArray type should be further constrained to `MGLAnnotationView *`, if that’s possible with the macros we’re using.